src/v2/Apps/Partner/Components/Overview/AboutPartner.tsx

```diff
@@ -46,11 +63,17 @@ export const AboutPartner: React.FC<AboutPartnerProps> = ({
         </Media>
 
         {canRenderWebsite && (
-          <RouterLink to={website!} target="_blank">
-            <Text mb={2} variant="sm">
-              {website}
-            </Text>
-          </RouterLink>
+          <Text
+            as="a"
+            mb={2}
+            variant="sm"
+            // @ts-ignore
+            href={website}
+            target="_blank"
+            onClick={() => tracking.trackEvent(tappedViewTrackingData)}
+          >
+            {website}
+          </Text>
```

Comment:
For reviewers/viewers: [Here](https://artsy.slack.com/archives/CP9P4KR35/p1655811697049969) you can find more context why `RouterLink` was replaced with `Text`

Reply:
that was really helpful link @dimatretyak thanks!